tests/test_simple.py

```diff
@@ -40,7 +40,7 @@ def setUp(self):
                 "register_password_auth_provider_callbacks",
             ]
         )
-        module_api.check_user_exists.return_value = make_awaitable(True)
+        module_api.check_user_exists.return_value = make_awaitable("@bob:test")
```

Comment:
This is a little bit hacky but I was unsure of how else to work this-essentially all but one of the tests rely on this value being truthy, so substituting a truthy value here seemed okay, and the other test required this mock to return what synapse would return, which is this value if the account exists (which is not what the test was testing and is out of the scope of this test) I thought it could be okay to use this value. Open to suggestions if this is too icky.

Reply:
I changed this to return the username like the other tests, I think that's cleaner.